uiServices/src/testCaseServices.go

```diff
@@ -283,12 +310,40 @@ func deleteTestCase(rw http.ResponseWriter, req *http.Request) {
 
 	err := os.Remove(filepath)
 	if err != nil {
-		logrus.Println("File was not deleted", err)
-		rw.WriteHeader(http.StatusInternalServerError)
+		logrus.Println("Error deleting the file", err)
+		rw.WriteHeader(http.StatusNoContent)
 		return
 
 	}
 
 	rw.WriteHeader(http.StatusNoContent)
 
 }
+func deleteAllTestCases(rw http.ResponseWriter, req *http.Request) {
+	testCasePathDir := getPathHeader(req)
+
+	if err := IsHeaderValid(testCasePathDir); err != nil {
+		logrus.Error("No heared path found", err)
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	files, err := ioutil.ReadDir(testCasePathDir)
+	if err != nil {
+		logrus.Error("Error reading the directory ", err)
+		rw.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	for _, file := range files {
+		filepath := fmt.Sprintf("%s%s", testCasePathDir, file.Name())
+
+		err := os.Remove(filepath)
+		if err != nil {
+			logrus.Error("Error removing the files from directory", err)
+			rw.WriteHeader(http.StatusNoContent)
```

Comment:
if you cannot delete the file, please return rw.WriteHeader(http.StatusInternalServerError)